source/agora/node/Validator.d

```diff
@@ -585,9 +585,9 @@ public class Validator : FullNode, API
             return Enrollment.init;
         }
 
-        log.trace("Sending Enrollment at height {} for {} cycles with {}",
-            next_height, this.params.ValidatorCycle, enroll_key);
         const enrollment = this.enroll_man.createEnrollment(enroll_key, avail_height);
+        log.trace("Sending Enrollment to be stored in pools for enrolling at height {} to participate in blocks {} to {}",
```

Comment:
```suggestion
        log.trace("Sending Enrollment for inclusion at height {} (validator range: [{} ; {}])",
```

IMO mentioning the pool assumes too much knowledge of the process.

Reply:
Updated but with slight change as the `validator range` could be misleading.